events/models.py

```diff
@@ -283,8 +286,11 @@ class RecurringRule(RuleMixin, models.Model):
     all_day = models.BooleanField(default=False)
 
     def __str__(self):
-        strftime = settings.SHORT_DATETIME_FORMAT
-        return f'{self.event.title} every {timedelta_nice_repr(self.interval)} since {date(self.dt_start.strftime, strftime)}'
+        return (
+            f'{self.event.title} every '
+            f'{timedelta_nice_repr(self.freq_interval_as_timedelta)} since '
+            f'{date(self.dt_start, settings.SHORT_DATETIME_FORMAT)}'
+        )
```

Comment:
```suggestion
        return (f'{self.event.title} every {timedelta_nice_repr(self.freq_interval_as_timedelta)} since '
                f'{date(self.dt_start, settings.SHORT_DATETIME_FORMAT)}')
```